crates/bevy_window/src/event.rs

```diff
@@ -236,6 +236,21 @@ pub struct WindowFocused {
     pub focused: bool,
 }
 
+/// An event that indicates a window is occluded.
```

Comment:
Using some synonyms here would really help clarify the docs.

Reply:
Ok, added the first part of the docs from winit as copy/paste and a link to the full docs in winit.